content/applications/inventory_and_mrp/manufacturing/workflows/subcontracting_dropship.rst

```diff
@@ -0,0 +1,263 @@
+
+   If there is stock on-hand, confirming an |SO| for the product will instead create a delivery
+   order, because Odoo assumes that the |SO| is fulfilled using the stock in the warehouse.
+
+   This is not the case for subcontracted products that are dropshipped to the end customer. In that
+   case, a *subcontractor* |PO| is **always** created, even if there is stock on-hand.
+
+Process subcontractor PO
+------------------------
+
+If a *subcontractor* |PO| was not created in the previous step, do so now by navigating to
+:menuselection:`Purchase app --> Orders --> Purchase Orders`, and clicking :guilabel:`New`.
+
+Begin filling out the |PO| by selecting a subcontractor from the :guilabel:`Vendor` drop-down menu.
+
+In the :guilabel:`Products` tab, click :guilabel:`Add a product` to create a new product line.
+Select a product produced by the subcontractor in the :guilabel:`Product` field, and enter the
+quantity in the :guilabel:`Quantity` field.
+
+Finally, click :guilabel:`Confirm Order` to confirm the *subcontractor* |PO|.
+
+When a |PO| is confirmed for a product that requires dropshipping components to a subcontractor, a
+receipt or dropship order is automatically created, and can be accessed from the corresponding
+:guilabel:`Receipt` or :guilabel:`Dropship` smart button that appears at the top of the |PO|.
+
+.. image:: subcontracting_dropship/subcontractor-po.png
+   :align: center
+   :alt: A subcontractor PO for a *Dropship Subcontractor on Order* product, with a Receipt smart
+         button at the top of the page.
+
+In addition, an |RfQ| is created for the components that are purchased from the vendor and sent to
+the subcontractor. However, the |RfQ| **IS NOT** automatically linked to the *subcontractor* |PO|.
+
+Once the |RfQ| is confirmed and becomes a *vendor* |PO|, a *Dropship Subcontractor* order is
+created. This order is linked to both the *vendor* |PO| and the *subcontractor* |PO|.
+
+Confirm vendor RfQ
+------------------
+
+To access the |RfQ| created by confirming the *subcontractor* |PO|, navigate to
+:menuselection:`Purchase app --> Orders --> Requests for Quotation`. Select the |RfQ| that lists the
+correct vendor in the :guilabel:`Vendor` field, and the reference number of the receipt that was
+created after confirming *subcontractor* |PO|, in the :guilabel:`Source Document` field.
+
+On the |RfQ|, the :guilabel:`Deliver To` field reads :guilabel:`Dropship Subcontractor`, and the
+:guilabel:`Dropship Address` field shows the name of the subcontractor to whom components are being
+dropshipped.
+
+Click :guilabel:`Confirm Order` to turn the |RfQ| into a *vendor* |PO|, and confirm the purchase of
+components from the vendor. After doing so, a :guilabel:`Dropship` smart button appears at the top
+of the *vendor* |PO|, and a :guilabel:`Resupply` smart button appears at the top of the
+*subcontractor* |PO|.
+
+.. image:: subcontracting_dropship/vendor-po.png
```

Comment:
Similar optional feedback here. Consider:
- highlighting **Deliver to** field (along with, in this case, the **Dropship Address**
- **image** --> **figure** directive